testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -4016,75 +4213,83 @@ QUnit.module('dxCalendar number and string value support', {
     });
 });
 
-QUnit.module('valueChanged handler should receive correct event', {
-    beforeEach: function() {
-        fx.off = true;
-        this.clock = sinon.useFakeTimers();
-        this.valueChangedHandler = sinon.stub();
-        this.$element = $('<div>')
-            .dxCalendar({
+['onValuesChanged', 'onValueChanged'].forEach((name) => {
```

Comment:
this test is too difficult to read for me now. Maybe it's better to keep it separate?